### ISSUE TITLE
Customise Ohana web search interface to work better for Lambeth

### ISSUE THREAD

**Issue by pmackay** [folklabs/connecting-communities#19]
The current default Ohana interface looks like this:

![screen shot 2015-05-15 at 13 38 21](https://cloud.githubusercontent.com/assets/115115/7652764/b6a9ced8-fb07-11e4-9527-e365cd52b289.png)

Possible changes that could be made to make it more appropriate for Lambeth:
- [ ] change/remove top left logo
- [ ] update About box
- [ ] update taxonomy terms
- [ ] remove Google Translate widget
- [ ] edit the footer text to be specific to LCC
- [ ] 

@djwesto can you add some styling suggestions please?

**Comment by djwesto** [comment]
- We probably need some sort of logo or identifying mark.
- Why not keep the Google Translate widget?
- Style-wise, what do you want? Colours and fonts? I'd leave the layout and wording as it is.

**Comment by pmackay** [comment]
How are we going to come up with a logo?

Could keep widget I suppose. Fair enough.

Yep, colours, fonts, any visual adjustments that make it look more distinct
and relevant to Lambeth.

**Comment by djwesto** [comment]
Not necessarily a logo – just needs a name which could stay in text. But if we want a logo then I could design one up quickly.

Are we doing this so it's relevant to Lambeth? I thought we were keeping it quite general and not focusing on Lambeth for the time-being?

**Comment by pmackay** [comment]
This is intended to be an index for Lambeth. And I think @GilesGibson was
keen.

**Comment by djwesto** [comment]
Fair enough then. Well, then a simple approach might be to try to find a suitable photograph of Lambeth, stick it in the top search area and adjust the typography and colour scheme to echo Lambeth Council's (without it actually being badged Lambeth Council directly).

**Comment by GilesGibson** [comment]
We could probably do with something that acknowledges Lambeth council. It does need to also say, more prominently "Connecting Communities".

**Comment by djwesto** [comment]
Don't want to seem rude but why does it actually need to say "Connecting Communities" anywhere? That's our project title but it's a real mouthful. Almost as bad as "Community Open Data" :stuck_out_tongue_winking_eye:

**Comment by GilesGibson** [comment]
so what would you call it? It may well be an interface for users, the hub of a major project etc. I was keen to show something that lambeth would recognise and appreciate.

**Comment by djwesto** [comment]
I'll have a think and try to brainstorm some names. Can't propose straight off. 

**"Connecting communities"** could well be a strapline that acknowledges Lambeth _and_ explains the _function_ of the system. But something shorter, snappier and more unique will be more likely to attract lots of users.

**Comment by GilesGibson** [comment]
Sure, but right now we need a test site to show people and the people who are mainly looking at it will know about the Connecting Communities name, hopefully. Something like "Ohana" can come later.

**Comment by djwesto** [comment]
Ah, so if we're just talking about the prototype site then CC will do – but **seriously** it's going to change for the live version. This time @djwesto's not taking no for an answer. :grinning:
